modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/ApplicationAPI.macro

```diff
@@ -31,6 +31,35 @@ definition {
 		return ${curl};
 	}
 
+	@summary = "Default summary"
+	macro addSchema(fieldName = null, fieldDescription = null, schemaName = null) {
```

Comment:
Thanks for creating the macro. There is already a `createAPISchema` macro in APIBuilderUI.macro file. We only add API related macros into ApplicationAPI.macro file, and will add UI related into APIBuilderUI.macro. Please remove the addSchema and update accordingly, thanks.

Reply:
Fixed, thanks! I did cherry-pick for this [commit](https://github.com/liferay-headless/liferay-portal/pull/1688/files#diff-2808598e91cea23a897873640952938ac3d189b17f7d9230fe40bc2065af2589) to use the macro edited.